tests/unit/ssg-module/test_oval_object_model/test_load_and_store.py

```diff
@@ -0,0 +1,323 @@
+import os
+import pytest
+import tempfile
+from ssg.xml import ElementTree
+
+from ssg.oval_object_model import load_oval_document, OVALDocument, OVALEntityProperty
+from ssg.oval_object_model.oval_entities import Criterion, ExtendDefinition
+
+from ssg.xml import open_xml, get_namespaces_from, register_namespaces
+
+
+DATADIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "data"))
+OVAL_DOCUMENT_PATH = os.path.join(DATADIR, "oval_with_correct_extend_definition.xml")
+
+PROJECT_ROOT = os.path.join(
+    os.path.dirname(__file__),
+    "..",
+    "..",
+    "..",
+    "..",
+)
+OVAL_DOCUMENT_PATH_FROM_BUILD = os.path.join(
+    PROJECT_ROOT, "build", "ssg-rhel9-oval.xml"
```

Comment:
What if a different product would be built? I think that a unit test should never depend on built artifacts.